x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -55,7 +54,7 @@ export function SpanFlyout({
   }
 
   const stackframes = span.span.stacktrace;
-  const codeLanguage: string = get(span, SERVICE_LANGUAGE_NAME);
+  const codeLanguage = idx(parentTransaction, _ => _.service.language.name);
```

Comment:
I would NEVER have spotted this without typescript: apparently spans no longer have `service.language.name`, and we now need to get it from the parent transaction.
I think @jalvz pointed it out when he helped me updated the typescript interfaces but I didn't connect the dots. 
TS <3